src/fetch-data/scrapUrl.js

```diff
@@ -30,7 +30,7 @@ export async function scrapUrl(url) {
       err = new Error(`Parsing Error: ${error.message}`);
     } else if (error instanceof got.HTTPError) {
       err = new Error(
-        `HTTP Error: ${error.response.statusCode} - ${error.options.url?.href} - ${error.message}`
+        `HTTP Error: ${error.response.statusCode} - ${error.options.url.href} - ${error.message}`
```

Comment:
```suggestion
        `HTTP Error: ${error.response?.statusCode} - ${error.options?.url?.href} - ${error.message}`
```

Reply:
its already node 14, I think `?` are not needed here since the error is provided by ` got`